libs/cart/src/facades/cart/cart-facade.interface.ts

```diff
@@ -30,6 +33,7 @@ export interface DaffCartFacadeInterface<
   grandTotal$: Observable<DaffCart['grand_total']>;
   coupons$: Observable<DaffCart['coupons']>;
   items$: Observable<DaffCart['items']>;
+  dictionaryOfItems$: Observable<Dictionary<U>>;
```

Comment:
I think `itemDictionary$` is a better name.

Reply:
agreed. changed